lib/tasks/advanced_search.rake

```diff
@@ -0,0 +1,11 @@
+require "publishing_api_finder_publisher"
+
+namespace :publishing_api do
+  desc "Publish advanced-search finder."
+  task :publish_advanced_search_finder do
+    finder = YAML.load_file("config/advanced-search.yml")
+    timestamp = Time.now.iso8601
+
+    PublishingApiFinderPublisher.new([{ file: finder, timestamp: timestamp }]).call
+  end
```

Comment:
`PublishingApiFinderPublisher` expects a logger as a potential second parameter, not a timestamp. The timestamp is already being calculated in the previous commit.

Reply:
The second (`:logger`) param is omitted here. The only param passed is an array of hashes (with one item in it). Mostly because this class was based on something similar from SpecialistPublisher which publishes multiple finders at once. Arguably this doesn't need to happen here.

Reply:
I'm going to dismiss my review, so that I'm not a blocker, but there are couple of things that I would change to make it a bit easier to follow:

You can't pass any parameters to the rake task, so I'd move the loading of the YAML file and setting of the default timestamp to `PublishingApiFinderPublisher`.

However, if you do still decide to pass a finder object from the rake task; are you expecting to publish multiple finders using `PublishingApiFinderPublisher`? If not, I'd change finder parameter from an array to a single finder. 
 
I'd also extract `finder[:file]` to make `PublishingApiFinderPublisher` easier to read.